resources.md

```diff
@@ -11,9 +11,48 @@
 <a href = "https://www.w3schools.com/css/"> W3School CSS tutorials </a> <br>
 <a href = "https://www.w3schools.com/bootstrap/default.asp"> W3School bootstrap tutorials</a> <br>
 <a href = "https://getbootstrap.com"> Bootstrap website </a> <br>
+<a href = "https://validator.w3.org/#validate_by_uri+with_options"> Online validator </a> <br>
+<a href = "https://stackoverflow.com/questions/8726843/should-i-be-using-absolute-font-sizes-small-medium-large-et-al"> Font sizes - SO </a> <br>
+
+
 
 # Books
 <a href = "http://www.htmlandcssbook.com"> HTML and CSS book</a> <br>
 <a href = ""> </a> <br>
 <a href = ""> </a> <br>
 
+# Articles
+<a href ="https://medium.com/wd-tips-tricks/essential-html-css-and-javascript-tools-and-libraries-bbd39b4ed7a"> Essential libraries</a> <br>
```

Comment:
We prefer to use Markdown Syntax for links: https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#links 😉